test/test-applications/integrations/TestApplication.Modules/TestApplication.Modules.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+    <Reference Include="System.Net.Http" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
```

Comment:
is it needed?

Reply:
Removed. https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1515/commits/42aebf63e7b2b5c18f66335bfb97729405d00203